common/src/main/java/com/wynntils/screens/maps/AbstractMapScreen.java

```diff
@@ -173,6 +174,11 @@ protected List<Poi> getRenderedPois(
                 if (iconPoi.getIconAlpha(currentZoom) < 0.1f) {
                     continue;
                 }
+            } else if (poi instanceof LabelPoi labelPoi) {
+                // Check if label is visible
+                if (labelPoi.getAlphaFromScale(currentZoom) < 0.1f) {
+                    continue;
+                }
```

Comment:
? Some branch mixup?

Reply:
Yes, it disappeared when you merged in the latest `gear-finished`.